torchgeo/datamodules/vaihingen.py

```diff
@@ -23,26 +27,44 @@ class Vaihingen2DDataModule(pl.LightningDataModule):
 
     def __init__(
         self,
-        batch_size: int = 64,
+        train_batch_size: int = 32,
         num_workers: int = 0,
+        patch_size: Tuple[int, int] = (64, 64),
```

Comment:
I know this is the same in `OSCDDataModule`, but personally I think it would be better to support both ints and tuples of ints. This also matches how our samplers work. Doesn't necessarily have to change in this PR, could be a follow-up PR.